ci/publish-oci.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+if [ ! -f "${HOME}/.unifi/bin/unifi" ]; then
+  echo "Unifi not found! Exit"
+  exit 1
+fi
+
+if [[ "$#" != 3 ]]; then
+ echo "No enjoy parameters!"
```

Comment:
I'm not sure if this is what you meant to write - maybe `Not enough parameters`?